include/engine/phantom_node.hpp

```diff
@@ -144,6 +151,7 @@ struct PhantomNode
         component.is_tiny = other.component.is_tiny;
 
         location = foot_point;
+        this->input_coordinate = input_coordinate;
```

Comment:
`foot_point` -> `location_`. `input_location` -> `input_location_`. This eliminates the `this->`.